test/cloudtest/pkg/commands/main.go

```diff
@@ -948,23 +985,42 @@ func (ctx *executionContext) createClusters() error {
 	}
 
 	for _, cl := range ctx.cloudTestConfig.Providers {
-		if ctx.arguments.onlyEnabled {
-			logrus.Infof("Disable cluster config:: %v since onlyEnabled is passed...", cl.Name)
-			cl.Enabled = false
+		enabledByCommandLine := contains(ctx.arguments.clusters, cl.Name)
```

Comment:
the method looks complexity, can extract private methods?

Reply:
I wonder why linter doesn't trigger. Probably because the signature of the method is not changed.

Reply:
> method createClusters looks complexity, can extract private methods?

do you mean to extract enable/disable logic?

Reply:
I meant, we need to refactor this method and expose new methods

Reply:
I've extracted enable/disable cluster logic to a separate function. Otherwise I'd prefer not to make changes to the code not related to  the PR subject.